lib/exportResume.js

```diff
@@ -71,8 +66,14 @@ function sendExportRequest(resumeJson, fileName, theme, format, callback) {
         })
         .set('Accept', 'application/json')
         .end(function(response) {
-            fs.writeFileSync(process.cwd() + '/' + fileName + format, response.text);
-            callback();
+		    var errorText = null,
+			    responseText = response.text;
+		    if(responseText && responseText.indexOf('<html>') > -1) {
```

Comment:
This is a rather strange method of error handling. Can you check if `.statusCode === 200` instead?